src/data-types/varbinary.ts

```diff
@@ -56,14 +56,42 @@ const VarBinary: { maximumLength: number } & DataType = {
   },
 
   generate: function* (parameter, options) {
-    if (parameter.value != null) {
-      const buffer = new WritableTrackingBuffer(0);
+    let value = parameter.value;
+
+    if (value != null) {
       if (parameter.length! <= this.maximumLength) {
-        buffer.writeUsVarbyte(parameter.value);
-      } else {
-        buffer.writePLPBody(parameter.value);
+        const buffer = new WritableTrackingBuffer(0);
+        buffer.writeUsVarbyte(value);
+        yield buffer.data;
+
+      } else { // writePLPBody
+        const UNKNOWN_PLP_LEN = Buffer.from([0xfe, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff]);
+
+        let length;
+        if (value instanceof Buffer) {
+          length = value.length;
+        } else {
+          value = value.toString();
+          length = Buffer.byteLength(value, 'ucs2');
+        }
+
+        yield UNKNOWN_PLP_LEN;
+        if (length > 0) {
+          const buffer = Buffer.alloc(4);
+          buffer.writeUInt32LE(length, 0);
+          yield buffer;
+
+          if (value instanceof Buffer) {
+            yield value;
+          } else {
+            yield Buffer.from(value, 'ucs2');
+          }
+        }
+
+        const end = Buffer.from([0x00, 0x00, 0x00, 0x00]);
```

Comment:
What do you think about making this a top-level constant too? Same reasoning as for https://github.com/tediousjs/tedious/pull/1049/files#r377817446. I think `PLP_TERMINATOR` would be a fitting name for this.